packages/invoices/src/InboundShipment/DetailView/SidePanel.tsx

```diff
@@ -19,9 +19,10 @@ import { InboundShipment } from '../../types';
 
 interface SidePanelProps {
   draft: InboundShipment;
+  update: (patch: Partial<InboundShipment>) => Promise<InboundShipment>;
```

Comment:
Instead of passing the update function (and even the draft) - at this point we could just `useQuery` and `useMutation` in these components and use the cached invoice value in all of these components. Just use a `useInbound(id)` and `useUpdateInvoice(id)` hooks and it would be quite easy

Reply:
that's a very neat idea!